src/main/java/io/confluent/connect/jdbc/source/JdbcSourceTask.java

```diff
@@ -243,14 +243,21 @@ public void stop() throws ConnectException {
 
         log.debug("Returning {} records for {}", results.size(), querier.toString());
         return results;
-      } catch (SQLException e) {
-        log.error("Failed to run query for table {}: {}", querier.toString(), e);
+      } catch (SQLException sqle) {
+        log.error("Failed to run query for table {}: {}", querier.toString(), sqle);
         resetAndRequeueHead(querier);
         return null;
+      } catch (Exception e) {
```

Comment:
Maybe `Throwable`?

Reply:
Yes, probably safer to catch Throwable here.  I'll change it.  Thanks for the review!